### DISCUSSION TITLE
create argument to Pluto.run to disable autosave

### DISCUSSION THREAD

**Discussion by adannenberg** [fonsp/Pluto.jl#2447] [Questions about using Pluto] [answered]
I know this has been discussed on Discourse but I'd like to re-raise the subject and present the use-case to justify the proposed change...  

I often have the same Dropbox or Google Drive-stored notebook open on both my laptop and my desktop.  To avoid losing changes made on the laptop (say), Pluto's autosave requires me to close and re-open the notebook on the desktop before making changes.  If I forget to do so then the more current version of the notebook that's on my laptop is lost.

I understand that all of this is really an argument for using git or some other vcs.  Yes, I agree.  But it would be nice if the simpler workflow of using a cloud-based filesystem could be made less risky by allowing the user to opt out of autosave (which also introduces the risk of losing your work - I get it).

**Comment by fonsp** [comment, marked as answer]
You can use `Pluto.run(disable_writing_notebook_files=true)` to ask Pluto to never write to any file. This means that nothing you do in this session will be saved to disk ⚠️ (even if our UI says that it is). The only way to persist work is to use the Export button.

Is this what you need?

**Reply by adannenberg** [reply]
Yes, that works for me.  Thank you for your help.

Parenthetically, the Pluto.run() entry on https://docs.juliahub.com/Pluto/OJqMt/0.14.5/autodocs/ includes "For the full list, see the [Pluto.Configuration](https://docs.juliahub.com/Pluto/OJqMt/0.14.5/autodocs/@ref) module." and the link is broken.  I did try... :D
